### PR TITLE
School library Refactor

### PR DESCRIPTION
**Pull request by codecaiine**
- I created this PR to check if our project was as requested.

- In this milestone, we had to refactor our project.

**Project requirements that are done:**
- Make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected ☑️
- Followed SOLID principles ☑️
- Make sure that the role of main.rb is only to start your code ☑️
- Extracted some pieces of logic into a separate class ☑️

### CODE REVIEW

File: module_book.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require './book'
```
Comment (D4nielJ) [inline module_book.rb]: - [x] Make sure to use `require_relative 'file'` every time you are importing relative files, this is a best practice on Ruby. [You can read more about it here.](https://rubystyle.guide/#use-require_relative-whenever-possible)

**Review by D4nielJ — APPROVED** [review]
## Hello @codecaiine and @Kossi-stack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: module_book.rb (hunk 1 above)
Comment (codecaiine) [reply module_book.rb]: Hey @D4nielJ 
Thank you.

**Comment by codecaiine** [comment]
Hi @D4nielJ 
Thank you for reviewing our work and approving it.

Cheers!

**Pull request merged**